src/cover-hero.tsx

```diff
@@ -0,0 +1,178 @@
+            description,
+            descriptionLinesMax,
+            extra,
+            sideExtra,
+            button,
+            secondaryButton,
+            buttonLink,
+            minHeight,
+            aspectRatio = 'auto',
+            variant,
+            backgroundColor,
+            centered,
+            noPaddingY,
+            ...mediaProps
+        },
+        ref
+    ) => {
+        const hasMedia = mediaProps.backgroundVideo || mediaProps.backgroundImage;
+
+        const background = hasMedia
+            ? 'none'
+            : backgroundColor ||
+              {
+                  default: vars.colors.background,
+                  inverse: vars.colors.backgroundBrand,
+                  alternative: vars.colors.backgroundAlternative,
+              }[variant ?? 'default'];
+
+        const textShadow = hasMedia ? '0 0 15px rgba(0, 0, 0, 0.4)' : undefined;
+
+        const mainContent = (
+            <div className={styles.mainContent}>
+                {headline && <Box paddingBottom={{desktop: 8, tablet: 8, mobile: 16}}>{headline}</Box>}
+                <Stack space={16}>
+                    <Stack space={8}>
+                        {pretitle && (
+                            <div className={styles.sixColumns}>
+                                <Text3 regular truncate={pretitleLinesMax} textShadow={textShadow}>
+                                    {pretitle}
+                                </Text3>
+                            </div>
+                        )}
+                        <Text8 as={titleAs} truncate={titleLinesMax} textShadow={textShadow}>
+                            {title}
+                        </Text8>
+                    </Stack>
+                    {description && (
+                        <div className={styles.sixColumns}>
+                            <Text3
+                                as="p"
+                                regular
+                                truncate={descriptionLinesMax}
+                                color={hasMedia ? vars.colors.textPrimary : vars.colors.textSecondary}
+                                textShadow={textShadow}
+                            >
+                                {description}
+                            </Text3>
+                        </div>
+                    )}
+                </Stack>
+                {extra}
+            </div>
+        );
+
+        return (
+            <section
+                ref={ref}
+                className={classnames(styles.coverHero, {
+                    [styles.centered]: centered,
+                    [styles.hasSideExtra]: sideExtra,
+                    [styles.minHeight]: !noPaddingY,
+                })}
+                style={{
+                    minHeight: aspectRatio && aspectRatio !== 'auto' ? 'auto' : minHeight,
+                    boxSizing: 'border-box',
+                    background,
+                    aspectRatio: aspectRatioToCssString(aspectRatio),
```

Comment:
aspectRatio is not supported by all the Chrome versions we need: https://caniuse.com/?search=aspect-ratio

Why not using [AspectRatioContainer](https://github.com/Telefonica/mistica-web/blob/master/src/utils/aspect-ratio-support.tsx#L47) (doing something similar to what we do with cards)

Reply:
I wanted to avoid the paddingTop hack because this is a rare case in this component and the graceful degradation for old browsers is quite good here.
Anyway, I finally included the hack, although I needed an extra div :(